src/sbt-test/caching/unzip/build.sbt

```diff
@@ -17,6 +24,14 @@ lazy val root = (project in file(".")).
       _.withCacheOutput(true)
         .withCacheUnzip(true)
         .withAssemblyUnzipDirectory(tempUnzipDir)
+        .withUseHardLinks(true)
+    },
+    logLevel in assembly := sbt.Level.Info,
+    extraLoggers in assembly ~= { currentFunction =>
```

Comment:
@eed3si9n do you see how this isn't working?  I just get a `List()`

Reply:
We rewrote logger / went back to using our own logger so this feature may or may not work any more actually.

Reply:
Thanks for the tip, found a work around!